packages/client/package.json

```diff
@@ -95,29 +94,27 @@
     "rxjs": "7.8.1",
     "styled-components": "5.3.9"
   },
+  "overrides": {
```

Comment:
OpenFin Workspace libs have dev dependencies with vulnerabilities exposed .. until they resolve, this will satisfy `npm audit` and dependabot